_includes/nav-links.html

```diff
@@ -0,0 +1,7 @@
+
+<!-- eventually this will be a logo -->
+<li><a href="/">Homepage</a></li>
```

Comment:
It might be easier for the future if we add a config.yml file that has a base site url, and use that in these links.